europe/README.EUROPE

```diff
@@ -1,245 +1,270 @@
+
+* DK
+* DK_bornholm.pol
+* DK_bridges.pol
+* DK_general.pol
+* DK_jutland.pol
+* DK_zealand.pol
+
+### The Faroe Islands: System definitions related to ETRS89
+
+*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Init file with various transformations and conversions of relevance in
+The Faroe Islands. All definitions in the init file uses the local ETRS89
+reference frame as a pivot datum. More information can be found in the
+[NordicTransformations](https://github.com/NordicGeodesy/NordicTransformations)
+repository.
+
+* FO
+
+### Nordic+Baltic countries: ITRFxx -> Local ETRS89 realizations
+
+*Source*: [The Nordic Geodetic Commission](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Transformations to and from the common Nordic referenc frame NKG_ETRF00.
+This init-file describes transformations between global reference frames
+and NKG_ETRF00 as well as transformations from NKG_ERTF00 to the local
+realisations of ETRS89 in each of the countries involved with NKG.
+
+All transformations in this init-file uses the common Nordic frame as a
+pivot datum. Exempt from this dogma are transformations with labels
+starting with ``_``. Those transformations are "private" to this file and are
+only used as steps in more complicated transformations.
+
+* NKG
+
+* CHENYX06.gsb
+* CHENYX06_etrs.gsb
+
+### Switzerland: LV03 (system CH1903) -> LV95 (system CH1903+)
+
+Grid transformation from Swiss reference frame LV03 (system CH1903) to Swiss reference frame LV95 (system CH1903+).
+
+*Source*: [swisstopo](https://www.swisstopo.admin.ch/en/knowledge-facts/surveying-geodesy/reference-frames/transformations-position.html)
+*Format*: NTv2.0 (.gsb)
+*License*: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/legalcode
)
+*Credit*: (c) 2007 swisstopo
+
+Recommended to transform from Swiss reference frame LV03 (system CH1903, EPSG:21781) to Swiss reference frame LV95 (system CH1903+, EPSG:2056). 
+LV95 is aligned to ETRF93, but shifted by several 100 meters.
+CH1903+ is the official reference system for federal surveying and for all national geobasis data regulated by federal law since 2017.
+CHENYX06.gsb gives corrections (distorsions between LV03 and LV95) in CH1903, CHENYX06_etrs.gsb gives the corrections in ETRS. 
+Several transformation services are available on https://www.swisstopo.admin.ch/en/maps-data-online/calculation-services.html (web forms and REST web geoservices) to support the exact transformation procedure between CH1903 and CH1903+ using the REFRAME algorithm (finite element transformation). The difference between the grid and finite element transformation is on a level of below several millimeters.
+
+Example:

```

Comment:
@kbevers I'm not sure we want to advertize those PROJ.4 era way of definining CRS and their transformations?

Reply:
No, that part should be left out. One thing is that the PROJ.4 style definitions. Another is that this is just an overview of the available grids, not a usage guide.

So Elmar, can you please remove the example section, so this follows the same format as the rest of the grid descriptions?

Reply:
done.